### DISCUSSION TITLE
I keep getting rejected because my first name and last name doesn't match my academic affiliation document

### DISCUSSION THREAD

**Discussion by kasdvn17** [community/community#137556] [GitHub Education]
### Hi everyone,

Hi everyone,
I have submitted over 5 requests for the student benefits. However, I got all of them rejected. The latest submission got rejected because my my first name and last name doesn't match my academic affiliation document.
<img width="346" alt="image" src="https://github.com/user-attachments/assets/031aa8ca-a862-472f-bed0-33577064e899">

I have updated my billing information as well as the public name displayed on my profile; nonetheless, nothing worked. I'm wondering whether because of different name format (I'm Vietnamese) and my full name has 4 words, so I got rejected.

Thanks for your help in advance.

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*
